network-api/networkapi/mozfest/models.py

```diff
@@ -53,6 +53,7 @@ class MozfestPrimaryPage(FoundationMetadataPageMixin, FoundationBannerInheritanc
             ("tabbed_profile_directory", customblocks.TabbedProfileDirectory()),
             ("newsletter_signup", customblocks.NewsletterSignupBlock()),
             ("stats_block", mozfest_blocks.StatisticsBlock()),
+            ("carousel_text_block", mozfest_blocks.CarouselTextBlock()),
```

Comment:
Looking at the existing block names above, what do you think about not including `_block` in the bock name?

Reply:
Yea that's a good point, honestly couldn't decide but that makes sense, even with the rebase drama it will cause it's worth doing :+1:

Reply:
Sorry, did not know rebase drama would follow 😅